runhouse/scripts/kubernetes_cluster/main.tf

```diff
@@ -0,0 +1,93 @@
+# Script to spin up a quick EKS cluster that can be used for testing Runhouse Kubernetes support
```

Comment:
can we add a README to this folder explainijng this sript, what needs to be changed, etc.?